<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatdlg</class>
 <widget class="QDialog" name="chatdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="wd_send" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(245, 245, 245);</string>
   </property>
   <widget class="QPushButton" name="pb_file">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_emo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>表情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_img">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>截图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_record">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_voc">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>语音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_vio">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_send">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 233, 233);color:rgb(32, 196, 110);</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_send">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="placeholderText">
     <string>请输入...</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="lw_chat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>501</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(245, 245, 245);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>501</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_chatMsg">
    <item>
     <widget class="QWidget" name="wd_msg" native="true">
      <property name="minimumSize">
       <size>
        <width>341</width>
        <height>41</height>
       </size>
      </property>
      <widget class="QPushButton" name="pb_mem">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lb_msg">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="wd_name" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(245, 245, 245);</string>
   </property>
   <widget class="QPushButton" name="pb_info">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>聊天界面名字</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
